src/controllers/connectionManager.ts

```diff
@@ -600,6 +601,35 @@ export default class ConnectionManager {
             // close active connection
             await this.disconnect(fileUri);
             // connect to the server/database
+            if (connectionCreds.authenticationType === Constants.azureMfa) {
+                if (!connectionCreds.azureAccountToken) {
+                    let account = this.accountStore.getAccount(connectionCreds.accountId);
+                    let profile = new ConnectionProfile();
```

Comment:
Do we not have a clone function on connection profile? That would be nice to add if not.